glaciers/anaconda-project.yml

```
@@ -18,11 +18,12 @@ channels:
 - defaults
 
 packages: &pkgs
-- python=3.8
-- bokeh
+- python=3.11
+- bokeh>= 3.3.4
```

Just to know, did you have to pin `bokeh >=3.3.4` to get the latest version?

Suggesting a small change to align with how we usually format version pins.

```suggestion
- bokeh >=3.3.4
```

---

Yeah. The reason I pinned `bokeh >=3.3.4` was because of the other time I had difficulty getting the latest bokeh version so you suggested I pinned it this way and it worked. So I just do that for all updates now.